### ISSUE TITLE
MATlink: MFunction and MScript doesn't work in Mathematica

### ISSUE THREAD

**Issue by hunghtd** [rsmenon/MATLink#77]
Hi everyone,

I'm using MATlink to call Matlab from Mathematica with my Macbook, but MFunction and MScript cannot run in Mathematica notebooks. Whenever I evaluate a notebook which contains either MFunction or MScript, Mathematica runs forever. Others work well such as MEvaluate, MSet, MGet... I'm using OSX verion 10.9.5, Matlab R2014a, and Mathematica 10. Moreover, everything is Ok with my iMac with the same version of OS, Matlab, and Mathematica.

Please advice me what I should fix this issue. Thanks a lot.

**Comment by rsmenon** [comment]
MATLAB 2014a has a bug that causes this issue on OS X and Linux. We have made a note of this in our [installation instructions](http://matlink.org/#installation) (see "warning"). Unfortunately, the only solution to this problem is to use an older version of MATLAB or the latest MATLAB R2014b (where the bug has been fixed).

I don't know why it worked in your iMac but my only guess is that you happen to have R2013a/b or R2014b installed.
